src/com/intellij/idea/plugin/hybris/project/utils/Plugin.kt

```diff
@@ -1,7 +1,6 @@
 /*
- * This file is part of "SAP Commerce Developers Toolset" plugin for Intellij IDEA.
- * Copyright (C) 2014-2016 Alexander Bartash <AlexanderBartash@gmail.com>
- * Copyright (C) 2019-2023 EPAM Systems <hybrisideaplugin@epam.com> and contributors
+ * This file is part of "SAP Commerce Developers Toolset" plugin for IntelliJ IDEA.
+ * Copyright (C) 2019-2024 EPAM Systems <hybrisideaplugin@epam.com> and contributors
```

Comment:
@mlytvyn Did I ask you not to do this?

Reply:
@AlexanderBartash , this is new class, GitHub shows incorrect changes.

Reply:
I am not going to argue you. Next time I see it, you will get DMCA'ed.

Reply:
compare files, there is nothing similar, from name to content.

Reply:
Okay, you started it.
You did not write this https://github.com/epam/sap-commerce-intellij-idea-plugin/blob/main/src/com/intellij/idea/plugin/hybris/project/descriptors/HybrisProjectDescriptor.java
You did not write this. So what we are DMCAing this repository?

Reply:
@AlexanderBartash , y're absolutely right, copyright for some classes was updated by mistake. Thank you for sharing a valid example, I'll update copyright for that file accordingly.

Reply:
That is just one example of many. It is your job to find them and fix if you want to keep having this repository.